Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -2896,6 +2928,667 @@ script:
+      description: A JSON string that describes the detection logic for the IOA exclusion.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.groups.id
+      description: Groups ID that the exclusion rule is associated with.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.groups.group_type
+      description: Groups type that the exclusion rule is associated with.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.groups.name
+      description: Groups name that the exclusion rule is associated with.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.groups.description
+      description: Groups description that the exclusion rule is associated with.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.groups.assignment_rule
+      description: Groups assignment rule that the exclusion is associated with.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.groups.created_by
+      description: Indicate who created the group.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.groups.created_timestamp
+      description: The date when the group was created.
+      type: Date
+    - contextPath: CrowdStrike.IOAExclusion.groups.modified_by
+      description: Indicate who last modified the group.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.groups.modified_timestamp
+      description: The date when the group was last modified.
+      type: Date
+    - contextPath: CrowdStrike.IOAExclusion.applied_globally
+      description: Whether the exclusion rule applies globally or only to specific entities.
+      type: Boolean
+    - contextPath: CrowdStrike.IOAExclusion.last_modified
+      description: The date when the exclusion rule was last modified.
+      type: Date
+    - contextPath: CrowdStrike.IOAExclusion.modified_by
+      description: Indicate who last modified the rule.
+      type: String
+    - contextPath: CrowdStrike.IOAExclusion.created_on
+      description: Indicate who created the rule.
+      type: Date
+    - contextPath: CrowdStrike.IOAExclusion.created_by
+      description: The date when the exclusion rule was created.
+      type: String
+  - arguments:
+    - description: A comma-separated list of quarantined files IDs to retrieve.
+      name: ids
+      required: false
+      isArray: true
+    - description: A custom filter by which the retrieve quarantined file should be filtered.
+      name: filter
+      required: false
+    - description: The SHA256 hash value of the quarantined files to retrieve.
+      name: sha256
+      required: false
+      isArray: true
+    - description: The file name of the quarantined files to retrieve.
+      name: filename
+      required: false
+      isArray: true
+    - description: Filter the retrieved files by state.
+      name: state
+      required: false
+    - description: Filter the retrieved files by hostname.
```

Comment:
If IsArray is true, then should this be: A comma-separated list of the hostnames of the files to retrieve.